api/tools/env/docker-compose.yml

```diff
@@ -1,90 +1,151 @@
+    ports:
+      - "55051:55000"
+    entrypoint:
+      - /scripts/entrypoint.sh
+      - wazuh-master
+      - worker1
+      - worker
+    environment:
+      - OPENDISTRO_SECURITY_ENABLED=false
+      - INDEXER_URL=https://wazuh-indexer:55000
 
-    wazuh-worker2:
-        image: dev-wazuh-manager
-        hostname: wazuh-worker2
-        volumes:
-            - ${WAZUH_LOCAL_PATH}/framework/scripts:/var/ossec/framework/scripts
-            - ${WAZUH_LOCAL_PATH}/api/scripts:/var/ossec/api/scripts
-            - ${WAZUH_LOCAL_PATH}/framework/wazuh:/var/ossec/framework/python/lib/python${WAZUH_PYTHON_VERSION}/site-packages/wazuh
-            - ${WAZUH_LOCAL_PATH}/api/api:/var/ossec/framework/python/lib/python${WAZUH_PYTHON_VERSION}/site-packages/api
-        depends_on:
-            - wazuh-master
-        ports:
-            - "55052:55000"
-        entrypoint:
-            - /scripts/entrypoint.sh
-            - wazuh-master
-            - worker2
-            - worker
+  wazuh-worker2:
+    image: dev-wazuh-manager
+    hostname: wazuh-worker2
+    volumes:
+      - ${WAZUH_LOCAL_PATH}/framework/scripts:/var/ossec/framework/scripts
+      - ${WAZUH_LOCAL_PATH}/api/scripts:/var/ossec/api/scripts
+      - ${WAZUH_LOCAL_PATH}/framework/wazuh:/var/ossec/framework/python/lib/python${WAZUH_PYTHON_VERSION}/site-packages/wazuh
+      - ${WAZUH_LOCAL_PATH}/api/api:/var/ossec/framework/python/lib/python${WAZUH_PYTHON_VERSION}/site-packages/api
+    depends_on:
+      - wazuh-master
+    ports:
+      - "55052:55000"
+    entrypoint:
+      - /scripts/entrypoint.sh
+      - wazuh-master
+      - worker2
+      - worker
+    environment:
+      - OPENDISTRO_SECURITY_ENABLED=false
+      - INDEXER_URL=https://wazuh-indexer:55000
 
-    nginx-lb:
-        build:
-            context: ./nginx-lb
-        image: dev-nginx-lb
-        restart: on-failure:5
-        entrypoint:
-            - /scripts/entrypoint.sh
-        depends_on:
-            - wazuh-master
-            - wazuh-worker1
-            - wazuh-worker2
+  nginx-lb:
+    build:
+      context: ./nginx-lb
+    image: dev-nginx-lb
+    restart: "on-failure:5"
+    entrypoint:
+      - /scripts/entrypoint.sh
+    depends_on:
+      - wazuh-master
+      - wazuh-worker1
+      - wazuh-worker2
 
-    wazuh-agent:
-        build:
-            context: ./wazuh-agent
-        image: dev-wazuh-agent
-        entrypoint:
-            - /scripts/entrypoint.sh
-            - nginx-lb
-            - ${WAZUH_VERSION}
-        depends_on:
-            - wazuh-master
-            - wazuh-worker1
-            - wazuh-worker2
-            - nginx-lb
+  wazuh-agent:
+    build:
+      context: ./wazuh-agent
+    image: dev-wazuh-agent
+    entrypoint:
+      - /scripts/entrypoint.sh
+      - nginx-lb
+      - ${WAZUH_VERSION}
+    depends_on:
+      - wazuh-master
+      - wazuh-worker1
+      - wazuh-worker2
+      - nginx-lb
+
+  wazuh-agent-1:
+    build:
+      context: ./wazuh-agent-from-repository
```

Comment:
This context does not exist. Please check if it was missing from staging during git commit